npm/css-src/sass/_drag-and-drop.scss

```diff
@@ -11,20 +11,25 @@
   transition: outline-offset 0.1s ease-in-out, background-color 0.1s linear;
 }
 
-.drag-and-drop__dropzone [type=file] {
+.drag-and-drop__input {
   display: none;
   left: 10.1em;
   position: relative;
 }
 
-.drag-and-drop__dropzone p {
+.drag-and-drop__hint-text {
   margin-bottom: 0;
   margin-right: 10px;
   padding-top: 7px;
 }
 
-.drag-and-drop__dropzone label {
+.drag-and-drop__button {
   display: inline-block;
   margin-bottom: 0;
   width: auto;
 }
+
+.drag-and-drop__success {
+  color: #00703c;
```

Comment:
Just noticed this is in the design system palette as `govuk-colour("green")`.

Reply:
Ah, Cool! I didn't know about that. Thanks, I've added it.